.dockerignore

```diff
@@ -0,0 +1,19 @@
+.git
+.gitignore
+.dockerignore
+Dockerfile
+
+
+# From .gitignore, manually maintained!
```

Comment:
replace with "append `.gitignore` to `.dockerignore` in travis `before_install`?:
```yml
before_install:
- cat .gitignore >> .dockerignore 
- docker build -t compat-travis-runner .
```